.gitignore

```diff
@@ -44,4 +44,8 @@ yarn.lock
 
 npm-debug.log*
 yarn-debug.log*
-yarn-error.log*
\ No newline at end of file
+yarn-error.log*
+
+#quickstart files
+examples/
```

Comment:
we have some directories named examples under `resources`, will this make git ignoring them?

Reply:
This only excludes examples in the root folder.

```
➜  incubator-pinot git:(ignore-quickstart-files) git status
On branch ignore-quickstart-files
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   pinot-tools/src/main/resources/examples/batch/airlineStats/hadoopIngestionJobSpec.yaml

```